tests/source/group_imports-merge_imports.rs

```diff
@@ -0,0 +1,17 @@
+// rustfmt-group_imports: StdExternalCrate
```

Comment:
Oh and forgot to mention this earlier, but may be worth moving these tests to a new `group_imports` directory under `tests/{source/target}/configs/` where the majority of config option specific tests are housed. That should help simplify the file names and it'd probably also be worth using the name of the variant in the file name too.